front/app/containers/Admin/projects/project/phase/PhaseHeader.tsx

```diff
@@ -228,9 +229,12 @@ export const PhaseHeader = ({ phase, tabs }: Props) => {
                 pathname,
                 url
               )}
-            >
-              <Link to={url}>{label}</Link>
-            </Tab>
+              statusLabel={
+                name === 'report' && !isExpired('01-04-2024') ? (
+                  <NewBadge ml="8px" />
```

Comment:
Why do you have two different badges for messaging and here?

Reply:
I don't know, also I think this label has been here for ages. I will ask if I can remove it